inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/rocks/inspectit/oce/core/config/ConfigurationCenter.java

```diff
@@ -0,0 +1,36 @@
+package rocks.inspectit.oce.core.rocks.inspectit.oce.core.config;
```

Comment:
somehow the directories and packages here are wrong for all the files in the `core`? can you check pls what went wrong?